src/model/page-config-resolver-test.js

```diff
@@ -300,6 +316,27 @@ describes.realWin('PageConfigResolver', {}, env => {
       });
     });
 
+    it('should handle alternate item types', () => {
+      const divElement = createElement(doc, 'div');
+      divElement.innerHTML =
+          '<div itemscope itemtype="http://schema.org/CreativeWork http://schema.org/Other"> \
+            <meta itemprop="isAccessibleForFree" content="True"/> \
+            <div itemprop="isPartOf" itemscope itemtype="http://schema.org/CreativeWork http://schema.org/Product"> \
+              <meta itemprop="name" content="New York Times"/> \
+              <meta itemprop="productID" content="pub1:premium"/> \
+            </div> \
+            <div itemprop="articleBody" class="paywalled-section"> \
+              Paid content possibly. \
+            </div> \
+          </div>';
+      addMicrodata(divElement);
+      const resolver = new PageConfigResolver(gd);
+      return resolver.resolveConfig().then(config => {
+        expect(config.isLocked()).to.be.false;
+        expect(config.getProductId()).to.equal('pub1:premium');
+      });
+    });
+
```

Comment:
could we have a negative test case for a number of types but none of them feature in the Allowed type list.

Reply:
done